model/job.go

```diff
@@ -225,6 +234,124 @@ func (j *Job) loadJobSpec() (err error) {
 	return nil
 }
 
+// MergeSpec is used to merge temporary spec patches into each job. otherJob should only be
+// the hcf/patch-properties job.  The code assumes package and property objects are immutable,
+// as they're now being shared across jobs. Also, the results are undefined if specified packages
+// or properties already exist in the destination job.
+func (j *Job) MergeSpec(otherJob *Job) {
+	// Ignore otherJob.Name, otherJob.Description
+	if otherJob.Packages != nil {
+		j.Packages = append(j.Packages, otherJob.Packages...)
+	}
+	// Skip templates -- they're only in place to keep `create-release` happy.
+	if otherJob.Properties != nil {
+		j.Properties = append(j.Properties, otherJob.Properties...)
+	}
+}
+
+// WriteConfigs merges the job's spec with the opinions and writes out json-encoding to the specified path.
```

Comment:
Wording is a bit awkward here. "… and writes out the JSON-encoded result to the specified path", perhaps? "… and writes out the result as JSON to the specified path"?